src/components/InventoryTable/InventoryTable.js

```diff
@@ -85,6 +85,7 @@ const InventoryTable = forwardRef(
       showCentosVersions = false,
       enableExport,
       lastSeenOverride,
+      axios,
```

Comment:
I am testing this in SSH env, should I pass the axios instance into the inventory table from the RHEL Advisor side?